src/views/RegisterForm/RegisterForm.jsx

```diff
@@ -78,15 +79,22 @@ export const RegisterForm = ({
       </Form>
 
       <div>
-        <h6>
-          Have an account?
+        <div className="form__seperator">
+          <hr className="form__horizontal__line"></hr>
+          Or
+          <hr className="form__horizontal__line"></hr>
+        </div>
+        <div>
+          <h6>
+            Have an account?
+          </h6>
           <Button
             className="form__toggle"
             onClick={setFormType}
           >
-            Login to your account.
+            {"👋 Log in to your account."}
```

Comment:
Remove period at end of sentence

Reply:
Hasn't been changed. Don't resolve it until updated